IopHomeNode.proto3

```diff
@@ -248,6 +248,12 @@ message ConversationResponse {
 //
 // An administrative message to test whether the peer is there and responds to requests.
 //
+// Each identity client is required to send a message not more than 60 seconds after the connection 
```

Comment:
Bikeshed moment... What would happen if we raise the timeout to 300s also on the identity clients? Would there be some resource used up too long on the home node? I understand that the chance of losing the connection to a mobile client is higher than to another home node, but is it a big problem if we do not see that immediately, especially if there is no traffic right now?

Battery usage might improve if the mobile clients would not need to send too many packets when their users are idle.